### PR TITLE
Adds filter-function to metalsmith

### PR DESCRIPTION
**Pull request by KaiVolland**
This adds a filter function to Metalsmith to avoid copying the spec-files to the examples folder.

### CODE REVIEW

File: webpack.examples.config.js [hunk 1]
```diff
@@ -78,27 +78,4 @@ commonConfig.module = {
...
-      warnings: false,
-      pure_getters: true,
-      unsafe: false,
-      screw_ie8: true
-    },
-    output: {
-      comments: false
-    }
-  })
-];
```
Comment (dnlkoch) [inline webpack.examples.config.js]: Why did you remove this?

Comment (marcjansen) [reply webpack.examples.config.js]: Yeah, was about to ask the same. without this the FeatureGrid example js clocks in at a smooth 4 MB:

image

Comment (KaiVolland) [reply webpack.examples.config.js]: I was just pissed that it took me 5 mins to build the examples.
I didn't check the impact of this.

Even if the FeatureGrid.js only reaches 826KB on my machine without uglifyJS we should not care how long it takes the ci to build the examples, so i readded it.

Maybe we can find a way to set these options environment specific.

Comment (marcjansen) [reply webpack.examples.config.js]: We basically have this already. During development we do not want to have the most optimized builds, only when we are building the examples which we want to publish.

The root problem is that the dev setup right now isn't perfect.
